spark/common/src/main/java/org/apache/spark/sql/execution/datasources/geoparquet/internal/ParquetColumnVector.java

```diff
@@ -31,11 +33,65 @@
 import org.apache.spark.sql.types.DataTypes;
 import org.apache.spark.sql.types.MapType;
 import org.apache.spark.sql.types.StructType;
+import org.apache.spark.sql.types.UserDefinedType;
 
 /**
  * Contains necessary information representing a Parquet column, either of primitive or nested type.
  */
 final class ParquetColumnVector {
```

Comment:
I am confused. This is introducing changes to files in `datasources/geoparquet` folder but your test case is against `parquet` data source. So looks like the change in this file is not used at all?